bcp47-orphans/library/Data/BCP47/Esqueleto.hs

```diff
@@ -1,4 +1,5 @@
-{-# OPTIONS_GHC -fno-warn-orphans #-}
+{-# OPTIONS_GHC -Wno-deprecations #-}
+{-# OPTIONS_GHC -Wno-orphans #-}
```

Comment:
Dumb question, does this downgrade these errors to be warnings?

Reply:
It's a bit of a nuanced interaction.

`-W{name}` enables a warning and `-Wno-{name}` disables it. These may be on by default, or on because of we use `-Weverything` or on because we compile with `--pedantic`. In any event, they're on unless we shut them off like this.

Additionally, `--pedantic` says to treat all warnings like errors, which is why we have to shut them off to get a Green build.